BE/src/main/java/com/sidedish4/codesquad/sidedish/web/dto/DetailDto.java

```diff
@@ -0,0 +1,36 @@
+package com.sidedish4.codesquad.sidedish.web.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter
+@NoArgsConstructor
+@ToString
+public class DetailDto {
```

Comment:
아래 `@Builder` 가 모든 필드를 필요로 하기에, 아래와 같이 생성자 선언을 바꿔보는 것을 추천드려요.

```suggestion
@Getter
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@Builder
@ToString
public class DetailDto {
```

이렇게 하면, `AllArgsConstructor` 를 `private` 으로 가리고, 이 생성자에 대해 `Builder` 를 생성하는 효과를 누리게 됩니다.